pyproject.toml

```diff
@@ -22,12 +22,12 @@ evg-api = "evergreen.cli.main:main"
 
 [tool.poetry.dependencies]
 python = "^3.6"
-Click = "^7.0"
-python-dateutil = "^2.8.1"
-PyYAML = "^5.1"
-requests = "^2.22.0"
-structlog = "^19.1.0"
-tenacity = "^5.0.4"
+Click = "^7"
```

Comment:
I've simplified these because we don't really care and have too specific dependencies for a library is causing problems.